js/DataService.js

```diff
@@ -8,13 +8,46 @@ angular.module('PathOfDamage')
     var int = parseInt(string);
     return isNaN(int) ? null : int;
   }
+  
+  function Row(key, value) {
+    this.enabled = true;
+    this.name = '';
+    this.value = null;
+    if (key !== undefined) {
```

Comment:
The encoder can already handle that case, it's actually just there because it seemed like bad practice to have unused properties floating around in objects. Is that just not that big of a deal in webdev land?